component/main.jsonnet

```
@@ -31,6 +32,9 @@ local ns_patch =
     }
   );
 
+local deployThanosObjStorage =
+  std.length(std.objectFields(params.thanosObjectStorage)) > 0;
```

```suggestion
  std.length(params.thanosObjectStorage) > 0;
```

`std.length` directly works on objects.